runtime/src/main/java/org/corfudb/runtime/exceptions/StaleTokenException.java

```diff
@@ -0,0 +1,16 @@
+package org.corfudb.runtime.exceptions;
+
+import lombok.Getter;
+
+/**
+ * Created by dmalkhi on 7/21/17.
+ */
+public class StaleTokenException extends RuntimeException {
+    @Getter
+    private long correctEpoch;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this "correctEpoch" field final. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1165)